packages/build-info/src/index.ts

```diff
@@ -0,0 +1 @@
+export { getBuildInfo } from './get-build-info.js'
```

Comment:
This is now the only public API surface we expose every change to this function signature is considered breaking. 

All other functions are not on the API surface (except the behaviour is changed)